Tools/cases_generator/tier2_abstract_generator.py

```diff
@@ -176,7 +176,7 @@ def generate_abstract_interpreter(
         if uop.name in abstract.uops:
             override = abstract.uops[uop.name]
             validate_uop(override, uop)
-        if uop.properties.tier_one_only:
+        if uop.properties.tier == 1 or "specializing" in uop.annotations:
```

Comment:
Hm, given that this change occurs in three different files, I would have solved this differently -- when the uop is created, if `specializing` is given, force `tier = 1`. Unless this affects metadata generation?